pkg/tracer-collection/tracer-collection.go

```diff
@@ -190,6 +219,9 @@ func (tc *TracerCollection) TracerExists(id string) bool {
 }
 
 func (tc *TracerCollection) Close() {
+	for _, fd := range tc.containerMntNsFds {
+		unix.Close(fd)
+	}
```

Comment:
```suggestion
	tc.mu.Lock()
	defer tc.mu.Unlock()
	for _, fd := range tc.containerMntNsFds {
		unix.Close(fd)
	}
	tc.containerMntNsFds = nil
```
To support multiple calls to Close().
And also take the mutex in case there is a container event during the Close() call.
I think it is cleaner.

Reply:
I'm wondering about `tc.containerMntNsFds = nil`, this will cause a crash on `TracerMapsUpdater` if there is a container event after `Close()`. Probably the right solution is to include a `closed` boolean, but it seems to much for me. WDYT?

Reply:
Fair enough.

Now that this function is non-empty, we should it in:
* pkg/local-gadget-manager/local-gadget-manager.go LocalGadgetManager.Close().
* pkg/gadgettracermanager/gadgettracermanager.go GadgetTracerManager.Close().

Reply:
Done!